packages/compass/src/app/index.js

```diff
@@ -185,110 +183,86 @@ var Application = View.extend({
       this.queryByHook('layout-container')
     );
 
-    const handleTour = () => {
-      if (app.preferences.showFeatureTour) {
-        this.showTour(false);
+    const handleTour = async () => {
+      const { showFeatureTour } = await preferencesIpc.getPreferences();
+      if (showFeatureTour) {
+        await this.showTour(false);
       } else {
         this.tourClosed();
       }
     };
 
     handleTour();
   },
-  showTour: function(force) {
+  showTour: async function(force) {
+    const { previousVersion } = await preferencesIpc.getPreferences();
     const TourView = require('./tour');
-    const tourView = new TourView({ force: force });
+    const tourView = new TourView({ force, previousVersion });
     if (tourView.features.length > 0) {
       tourView.on('close', this.tourClosed.bind(this));
       this.renderSubview(tourView, this.queryByHook('tour-container'));
     } else {
       this.tourClosed();
     }
   },
-  tourClosed: function() {
-    app.preferences.unset('showFeatureTour');
```

Comment:
Where is the unsetting happening now?

Reply:
With one of recent commits I replied showFeatureTour with the previousVersion session value and the lastKnownVersion preference.